app/models/activity_insight_oa_file.rb

```diff
@@ -3,4 +3,7 @@
 class ActivityInsightOAFile < ApplicationRecord
   belongs_to :publication,
              inverse_of: :activity_insight_oa_files
+
+  scope :pub_without_permissions, -> { left_outer_joins(:publication).where(publication: { licence: nil }).where(publication: { doi_verified: true }) }
+  scope :needs_permissions_check, -> { pub_without_permissions.where(version_checked: nil).where(%{version = 'acceptedVersion' OR version = 'publishedVersion'}) }
```

Comment:
Sorry, I didn't catch this the last time.  Can we do this like this:
```suggestion
  scope :needs_permissions_check, -> { pub_without_permissions.where(version_checked: nil).where(%{version = ? OR version = ?}, I18n.t('file_versions.accepted_version'), I18n.t('file_versions.published_version'))}
```